unitex/src/fr/umlv/unitex/frames/LemmatizeFrame.java

```diff
@@ -238,7 +238,7 @@ public void actionPerformed(ActionEvent e) {
 				 * right click
 				 */
 				int index=getBoxToSelectIndex(lemma,matchStartInTokens);
-				graphicalZone.getTaggingModel().selectBox(index);
+				graphicalZone.getTaggingModel().preferBox(index);
```

Comment:
Is this `preferBox` or `preferredBox`?

Reply:
This is preferBox because it gives status preferred to the box

Reply:
Understood !